### DISCUSSION TITLE
Bootleggers back to main org

### DISCUSSION THREAD

**Discussion by romgharti** [ROM-builders/temporary#17675] [General]
#17673
https://github.com/BootleggersROM/manifest
https://t.me/keepthebootleg/394303

**Comment by Apon77** [comment]
incomplete pr and still not stable to merge!

**Reply by romgharti** [reply]
> incomplete pr and still not stable to merge!

#17676 here is new,  sorry, incomplete pr and source already stable sir.. @Apon77
